go.mod

```diff
@@ -57,12 +57,17 @@ require (
 	go.uber.org/zap v1.24.0
 	golang.org/x/exp v0.0.0-20230711005742-c3f37128e5a4
 	golang.org/x/text v0.13.0
-	golang.org/x/time v0.1.0
-	golang.org/x/tools v0.6.0
-	google.golang.org/grpc v1.54.0
+	golang.org/x/time v0.3.0
+	golang.org/x/tools v0.10.0
+	google.golang.org/grpc v1.59.0
 	gotest.tools/gotestsum v1.7.0
 )
 
+require (
+	google.golang.org/genproto/googleapis/api v0.0.0-20231002182017-d307bd883b97 // indirect
```

Comment:
need format

Reply:
This is already formatted by `go mod tidy`

Reply:
done